NFT-Marketplace-Part-2.md

```diff
@@ -373,7 +377,7 @@ export function handleListingUpdated(event: ListingUpdated): void {
 }
 ```
 
-Awesome! Now, let's do `handleListingCanceled`. We don't want to display canceled listings on the marketplace, so we can just delete the entity from the datastore entirely.
+Awesome! Now, let's do `handleListingCanceled`. We don't want to display canceled listings on the marketplace, so we can just delete the entity from the datastore entirely. Insert the following code into the function.
 
 ```typescript
 export function handleListingCanceled(event: ListingCanceled): void {
```

Comment:
`ListingCanceled` should be changed to using 2 l's here as well

Reply:
Sorry, it should be good now.